plugins/rts54hub/rts54hub.quirk

```diff
@@ -27,13 +27,14 @@ Rts54RegisterAddrLen = 0x04
 Plugin = rts54hub
 GType = FuRts54HubDevice
 FirmwareSizeMin = 0x10000
-FirmwareSizeMax = 0x70000
+FirmwareSizeMax = 0x40000
```

Comment:
so smaller?

Reply:
yes, now our gen1 and gen2 USB Hub  FW size 64KByte  128KByte 192KByte(Rarely) and 256KByte(Rarely)